clusters/build-clusters/common/assets/admin_github-ldap-user-group-creator_rbac.yaml

```diff
@@ -0,0 +1,36 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    openshift.io/description: This SA is used to run github-ldap-user-group-creator
+  name: github-ldap-user-group-creator
+  namespace: ci
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: github-ldap-user-group-creator
+rules:
+- apiGroups:
+  - user.openshift.io
+  resources:
+  - groups
+  verbs:
+  - get
+  - list
```

Comment:
Says `creator` but can only read?

Reply:
I am running the dryRun mode on my laptop.
Will `--dry-run=false` after some code review.

Added `create` but commented for now. Will uncomment together with `delete`.